packages/grid/_modules_/grid/components/cell/GridCell.tsx

```diff
@@ -67,6 +67,7 @@ export const GridCell = React.memo(function GridCell(props: GridCellProps) {
       }
 
       const params = apiRef!.current.getCellParams(rowId, field || '');
+      delete params.api;
```

Comment:
Could we do the opposite? Only provide the api where it's absolutely needed?

Reply:
Will work to create different interfaces for where the `api` is needed.